ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -66,7 +66,7 @@
     - role: ccache
       when: ansible_distribution != "Solaris" # Compile fails on Solaris
     - role: nasm                  # OpenJ9
-      when: ansible_architecture == 'x86_64'
+      when: ansible_architecture == 'x86_z64'
```

Comment:
I'm assuming this was done due to https://adoptopenjdk.slack.com/archives/C53GHCXL4/p1614088449011500 ?
Either way, I think the cert has been renewed now (judging from the padlock in my browser).

Reply:
Yes. It was in a commit that had `DONOTMERGEME` in the title

Reply:
Ah okay- I think that was overwritten by your force pushes